src/main/java/ActionHandler.java

```diff
@@ -0,0 +1,66 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+
+public class ActionHandler {
+    enum allowedTask { todo , event , deadline};
```

Comment:
You missed out a space between the closing curly brace, and the character before it. Also, it is a good practice to have multi-line declaration, instead of having it as a one-liner. This improves code readability. Also, enums should be in caps, e.g. TODO, EVENT, DEADLINE